### PR TITLE
updated `Japanese`

### PR DESCRIPTION
**Pull request by riolove1**

### CODE REVIEW

**Comment by kitadai31** [comment]
Shouldn't contributions to translations be done through official ReVanced Manager instead of RVX Manager?
(I think you should contribute translations to official Revanced Manager instead of RVX Manager.)

Because RVX Manager is provided simply for inotia's patch compatibility.
If the translations are changed on ReVanced Manager side, the changes will also be reflected on RVX Manager.
Also, if you update a translation on the RVX Manager side, but the translation is also updated on the ReVanced Manager side, I think your changes will be overwritten by rebasing the base version of ReVanced Manager.

In my opinion, the correct way is to PR to RVX Manager repo for only strings specific to RVX Manager, such as "Remove unused library" feature.
Then other strings should be translated in the official ReVanced's Crowdin.

@inotia00 Excuse me, how is your opinion?
Because there are also other languages pull requests.

File: assets/i18n/strings_ja_JP.i18n.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "okButton": "OK",
   "cancelButton": "キャンセル",
-  "dismissButton": "取り消し",
+  "dismissButton": "取り消す",
   "quitButton": "終了",
   "updateButton": "更新",
-  "suggested": "推奨: ${version}",
+  "suggested": "推奨バージョン: ${version}",
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:7]: 推奨バージョンが画面からはみ出すので、そのままにすべきです

File: assets/i18n/strings_ja_JP.i18n.json [hunk 2]
```diff
@@ -23,16 +23,19 @@
...
+    "patchedSubtitle": "インストール済みのアプリ",
     "changeLaterSubtitle": "この設定は後から変更できます",
+    "noSavedAppFound": "アプリが見つかりません",
     "noInstallations": "パッチ済みのアプリはインストールされていません",
     "installUpdate": "更新を適用しますか？",
-    "updateSheetTitle": "RVX Managerを更新",
+    "updateSheetTitle": "RVX Manager を更新",
     "updateDialogTitle": "新しいアップデートが利用可能です",
-    "updatePatchesSheetTitle": "ReVanced Patchesを更新",
+    "updatePatchesSheetTitle": "ReVancedパッチを更新",
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:34]: 固有名詞なので変更すべきではありません

File: assets/i18n/strings_ja_JP.i18n.json [hunk 3]
```diff
@@ -77,10 +81,10 @@
     "storageButton": "APKファイルを選択",
     "selectFromStorageButton": "ストレージから選択",
     "errorMessage": "選択したアプリは使用できません",
-    "downloadToast": "ダウンロード機能は現在開発中です",
+    "downloadToast": "現在、ダウンロード機能は利用できません",
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:84]: これは原文が曖昧なのが悪いのですが、「現在、ダウンロード機能は利用できません」だと、この機能が一時的にダウンしているような印象を受けます。
ダウンロード機能は開発中でそもそも存在しないので、公式ManagerのCrowdinの方で私が意訳であえてこのようにしたのですが、できればそのままの方がいいと思います

File: assets/i18n/strings_ja_JP.i18n.json [hunk 4]
```diff
@@ -77,10 +81,10 @@
...
     "requireSuggestedAppVersionDialogText": "選択したアプリのバージョンは推奨バージョンではありません。予期せぬ問題が発生する可能性があります。\n推奨バージョンを選択してください。\n\n選択されたバージョン ${selected}\n推奨バージョン ${suggested}\n\n選択されたバージョンを使用する場合、設定から「推奨バージョンの使用を強制」を無効にしてください。",
     "featureNotAvailable": "この機能は未実装です",
-    "featureNotAvailableText": "このアプリは分割APKであり、パッチの適用とインストールにはroot権限でマウントする必要があります。ただし、ストレージから単一APKを選択することでパッチの適用とインストールが可能です。"
+    "featureNotAvailableText": "このアプリは分割された APK であり、root 権限でマウントすることによってのみ確実にパッチを適用してインストールできます。 ただし、ストレージから選択することで完全な APK をパッチしてインストールすることができます。"
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:87]: full APKの和訳は単一APKの方が適切だと思います
「ただし、ストレージから選択することで完全な APK をパッチしてインストールすることができます。」は前までの機械翻訳の文字列ですが、何を言いたいのかが伝わりにくいので戻さないで欲しいです

File: assets/i18n/strings_ja_JP.i18n.json [hunk 5]
```diff
@@ -138,52 +146,55 @@
...
-    "sourcesUpdateNote": "注: ReVanced PatchesとReVanced Integrationsを代替ソースから自動的にダウンロードします。\n\nこれにより、代替ソースとの通信が発生します。",
+    "sourcesUpdateNote": "注: ReVanced Patches と ReVanced Integrations を別のソースから自動的にダウンロードします。\n\nこれにより、他のソースに接続されます。",
     "apiURLLabel": "API の URL",
-    "apiURLHint": "RVX ManagerのAPIのURLを設定する",
+    "apiURLHint": "RVX Manager の API URL を設定します",
     "selectApiURL": "API の URL",
-    "orgPatchesLabel": "Patches の組織",
-    "sourcesPatchesLabel": "Patches のソース",
+    "orgPatchesLabel": "パッチの組織",
+    "sourcesPatchesLabel": "パッチのソース",
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:172]: 固有名詞なので変更すべきではありません

File: assets/i18n/strings_ja_JP.i18n.json [hunk 6]
```diff
@@ -201,15 +212,15 @@
...
     "resetStoredPatches": "パッチの選択をリセットしました",
     "resetStoredOptionsLabel": "パッチオプションをリセット",
-    "resetStoredOptionsHint": "すべてのパッチオプションをリセットする",
-    "resetStoredOptionsDialogTitle": "パッチオプションをリセットしますか？",
-    "resetStoredOptionsDialogText": "パッチオプションをリセットすると、保存されたすべてのオプションが削除されます。",
-    "resetStoredOptions": "オプションをリセットしました",
+    "resetStoredOptionsHint": "すべてのパッチ設定をリセット",
+    "resetStoredOptionsDialogTitle": "パッチ設定をリセットしますか？",
+    "resetStoredOptionsDialogText": "パッチ設定をリセットすると、保存されたすべてのオプションが削除されます。",
+    "resetStoredOptions": "設定をリセットしました",
```
Comment (kitadai31) [inline assets/i18n/strings_ja_JP.i18n.json:223]: 「パッチ設定」と「パッチオプション」が混在してしまうので、「パッチオプション」で統一した方がいいと思います。

**Review by kitadai31 — COMMENTED** [review]
I'll leave some reviews in case you decide to translate in the official ReVanced Manager's Crowdin.

**Comment by inotia00** [comment]
> Shouldn't contributions to translations be done through official ReVanced Manager instead of RVX Manager? (I think you should contribute translations to official Revanced Manager instead of RVX Manager.)
> 
> Because RVX Manager is provided simply for inotia's patch compatibility. If the translations are changed on ReVanced Manager side, the changes will also be reflected on RVX Manager. Also, if you update a translation on the RVX Manager side, but the translation is also updated on the ReVanced Manager side, I think your changes will be overwritten by rebasing the base version of ReVanced Manager.

I totally agree with this opinion.

This is also why I am not creating a Crowdin project for RVX Manager.

The method I currently follow is:
1. Fork the official ReVanced branch. This contains the latest translation changes.
2. Compare and merge the contributed translations via PR branches. Since RVX Manager has no other way to contribute translations than opening a PR, PRs are preferred if possible.
3. In case of translations with conflicting opinions, such as this PR, the merge is held back.

**Comment by riolove1** [comment]
@kitadai31 ご丁寧なご指摘ありがとうございます

**Pull request closed without merging**